### DISCUSSION TITLE
Laravel Jetstream/Breeze Compatibility

### DISCUSSION THREAD

**Discussion by wsdnathan** [orchidsoftware/platform#1546] [Q&A]
I am confused by the documentation which states:
> It is possible to disable the supplied authorization form altogether and make your own, for example, using the package Jetstream. `'auth' => true,`

Is there more information available or an example of this? Would Orchid still be able to mange users/roles/permissions or is it assumed that Fortify would be managed separately? Is it possible to use Orchid auth for admin users, and additionally use Breeze at the same time for frontend users?

**I am hopeful that the Orchid platform will be forward thinking to ensure support for the upcoming Spark 2 / spark-next.**

**Comment by tabuna** [comment]
Hey @wsdnathan. This setting will only disable the login page. Roles and other things have nothing to do with this setting. You can use any of the laravel packages for authorization.

**Reply by wsdnathan** [reply]
Am I correct in that by default it is simply using Laravel Auth and not too much else? If you wanted to install the new Spark (which is available now) then would you also want to install this? https://github.com/orchidsoftware/fortify I actually had not even seen that until now. If there was some documentation on how to bridge the gap between Laravel Breeze and Orchid it would be helpful to me.

Would Breeze frontend users be in the same database table as Orchid admin users? How could you provide distinction or separate the models or database tables? I'm interested in using Orchid as the backend for a CMS but am still confused on how it will interact with these Fortify derived packages like Jetstream etc.

**Reply by simonbuehler** [reply]
@wsdnathan did you find out more and if, whats the essence of it?

**Reply by wsdnathan** [reply]
No I have not looked further into it. If you have the correct permissions from Orchid configured into any of the Fortify/Breeze/Jetstream packages then I assume it will work as expected.

**Reply by tabuna** [reply]
You can install any system. Here's an example of our blog post where I show the Jetstream installation https://blog.orchid.software/authenticate-with-jetstream/
